BraveWallet/Panels/WalletPanelView.swift

```diff
@@ -283,11 +297,7 @@ struct WalletPanelView: View {
     }
     .foregroundColor(.white)
     .background(
-      LinearGradient(
-        colors: [.green, .blue],
-        startPoint: .top,
-        endPoint: .bottom
-      )
+      Blockies(seed: keyringStore.selectedAccount.id).walletPanelBackground()
```

Comment:
Can you form a similar wrapper around this like `Blockie` does? So that it takes an address instead of a seed and returns the background without having to call `walletPanelBackground`?

Something like `BlockieBackgroundGradient` or `BlockieMaterial` (like SwiftUI blur styles).

Only because it looks like `Blockies` here is a View but its not